language/stdlib/modules/LibraAccount.move

```diff
@@ -1102,6 +1139,29 @@ module LibraAccount {
         )
     }
 
+    /// The prologue for WriteSet transaction
+    fun writeset_prologue(
```

Comment:
The code in the old module had a (very simple) spec for the prologue function. Should that be pulled over here as well?